pkg/handler/project/project.go

```diff
@@ -309,6 +309,91 @@ func (h *handler) GetMembers(c *gin.Context) {
 		&view.PaginationResponse{Pagination: query.Pagination, Total: total}, nil, nil))
 }
 
+// DeleteMember godoc
+// @Summary Delete member in a project
+// @Description Delete member in a project
+// @Tags Project
+// @Accept  json
+// @Produce  json
+// @Param Authorization header string true "jwt token"
+// @Param id path string true "member ID"
+// @Success 200 {object} view.MessageResponse
+// @Failure 400 {object} view.ErrorResponse
+// @Failure 404 {object} view.ErrorResponse
+// @Failure 500 {object} view.ErrorResponse
+// @Router /project/:id/members [delete]
+func (h *handler) DeleteMember(c *gin.Context) {
+	memberID := c.Param("id")
+
+	// TODO: can we move this to middleware ?
+	l := h.logger.Fields(logger.Fields{
+		"handler":   "project",
+		"method":    "DeleteMember",
+		"projectID": memberID,
+	})
+
+	// get member infor
+	projectMember, err := h.store.ProjectMember.One(h.repo.DB(), memberID)
```

Comment:
What if an employee participates in many projects during the same period?

Reply:
Because we delete in project member view so I take ID of record in this table(project_members) as a input so it's unique